config/utils/options.js

```diff
@@ -98,23 +98,28 @@ const filterOptions = [
 
 const scopeOptions = [
   {
-    value: "day",
+    value: "today",
```

Comment:
Since `scopeOptions` and its values are used by a lot of different places in the codebase. I decided to add a 
 new key: `valueForApi`.